net/mwan3/files/lib/mwan3/mwan3.sh

```diff
@@ -37,89 +38,163 @@ MM_UNREACHABLE=""
 command -v ip6tables > /dev/null
 NO_IPV6=$?
 
-# return true(=0) if has any mwan3 interface enabled
-# otherwise return false
-mwan3_rtmon_ipv4()
+SCRIPTNAME="$(basename "$0")"
+LOG()
```

Comment:
We could also move the LOG function to common.sh.
So we have them in each mwan3 script if we include /lib/mwan3/common.sh

Reply:
Yes, good idea. Then we don't need to redefine it in mwan3track.